exercises/ruby-1-a/ruby_1_a_test.rb

```diff
@@ -1,33 +1,20 @@
 require 'minitest/autorun'
-require_relative 'ruby_1_a'
+require_relative 'ruby_1_a_example_3'
```

Comment:
This should be reverted (probably a remnant of you testing the third example :))

```suggestion
require_relative 'ruby_1_a'
```